kuma/spam/models.py

```diff
@@ -0,0 +1,18 @@
+from django.conf import settings
+from django.db import models
+from django.utils.translation import ugettext_lazy as _
```

Comment:
I'm fairly strongly opinionated about `_` being an alias for the lazy version of the gettext function since it works for clases – both import time (e.g. class attributes like model fields or in views classes) and process time (e.g. inside of view functions). It's a best practice that I've adopted from Django and other projects I've worked on. If – *and only if* – the non-lazy version of gettext is needed it's good to import it explicitly. I consider the current way how Mozilla project use "_lazy" to be an antipattern.